spec/i18n_spec.rb

```diff
@@ -213,6 +221,15 @@ def allowed_untranslated_key?(locale, key)
             )
           end
         end
+
+        it 'does not contain commonly misspelled words' do
+          flattened_yaml_data.each do |key, value|
+            locale = key.split('.', 2).first.to_sym
+            if COMMONLY_MISSPELLED_WORDS.key?(locale)
```

Comment:
Since the whole `describe` block is about testing an individual locale's file, I'd wonder if we could avoid running the spec at all for locales we're not interested in testing, to avoid evaluating and iterating through each key of the French and Spanish files. Maybe an early return (`next`?) before the loop, or a condition around the whole `it` block, or [RSpec conditional metadata](https://rspec.info/features/3-12/rspec-core/filtering/if-and-unless/).